src/service/stamp/index.ts

```diff
@@ -23,7 +23,10 @@ export async function pickStamp(beeDebug: BeeDebug, console: CommandLog): Promis
     exit(1)
   }
 
-  const choices = stamps.map(stamp => `${stamp.batchID} (${stamp.usageText})`)
+  const filterChoices = stamps.filter(stamp => stamp.batchTTL > 0)
+  const choices = filterChoices.map(
```

Comment:
I'd rather not create a `filterChoices` variable, instead just chain the functionals:

```
const choices = stamps.filter(x => x.batchTTL > 0)
    .map(/* ... */)
```

But if you want to keep it as a separate variable I'd suggest a better name such as `activeStamps`